addons/mail/static/tests/thread/thread_tests.js

```diff
@@ -994,10 +989,10 @@ QUnit.test("can be marked as read while loading", async function () {
             }
         },
     });
-    openDiscuss(undefined, { waitUntilMessagesLoaded: false });
+    openDiscuss(undefined);
```

Comment:
`openDiscuss(undefined|null)` can simply be `openDiscuss()` (nullish value doesn't matter)